### ISSUE TITLE
Accessibility: Missing "skip to content" link

### ISSUE THREAD

**Issue by MatzeKitt** [jhaals/yopass#3470]
According to Web Content Accessibility Guidelines (WCAG), a "skip to content" link is mandatory.

See: https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks

**Comment by MatzeKitt** [comment]
Not stale.

**Comment by jhaals** [comment]
Addressed
